adapter/internal/adapter/adapter.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/wso2/product-microgateway/adapter/internal/auth"
 	enforcerCallbacks "github.com/wso2/product-microgateway/adapter/internal/discovery/xds/enforcercallbacks"
 	routercb "github.com/wso2/product-microgateway/adapter/internal/discovery/xds/routercallbacks"
+	"github.com/wso2/product-microgateway/adapter/internal/ga"
```

Comment:
Shall we keep the name as **global-adapter**

Reply:
https://golang.org/doc/effective_go#package-names